utils/src/main/kotlin/LicenseFilenamePatterns.kt

```diff
@@ -61,5 +61,16 @@ data class LicenseFilenamePatterns(
                 "readme*"
             )
         )
+
+        private var instance: LicenseFilenamePatterns = DEFAULT
+
+        @Synchronized
+        fun configure(patterns: LicenseFilenamePatterns) {
+            log.info { "Configuring the license file patterns." }
+            instance = patterns
+        }
+
+        @Synchronized
+        fun instance(): LicenseFilenamePatterns = instance
```

Comment:
Commit message: You say ``getInstance()``` which has one trailing backtick too much, and shouldn't this function then also actually be called `getInstance()`?

Reply:
The commit message was intentionally just roughly speaking.

Reply:
Striped the backtick and the get.

Reply:
> The commit message was intentionally just roughly speaking.

But it was actually right in calling the function `getInstance()` IMO, as that's what's used in the current code base. I.e., the chance should have been in exactly the opposite direction.

Reply:
That was not clear to me from your previous comment.
I'll change it if and only if you press the request change button.
...Because I believe it's good enough ;-)

Reply:
To quote myself:

> shouldn't this function then also actually be called `getInstance()`?

In my view this sentence makes it very clear that *if* some alignment is made, I'd expect it to be on `getInstance()`, not `instance()`.

> I'll change it if and only if you press the request change button.

I'll do that.

Reply:
I interpreted the "then" as follows: "Because the commit message says getInstance() the code should also be getInstance().

Reply:
Anyhow, I amended the commit finally...as the rebase conflicts I expected didn't happen, they are in my other PR.